website_docs/automatic_instrumentation.md

```diff
@@ -0,0 +1,71 @@
+---
+Title: "Automatic Instrumentation"
+Weight: 3
+---
+
+Automatic instrumentation with Java uses a Java agent JAR that can be attached to any Java 8+ application. It dynamically injects bytecode to capture telemetry from many popular libraries and frameworks. It is the preferred way to capture telemetry data at the "edges" of an app or service, such as inbound requests, outbound HTTP calls, database calls, and so on. To instrument application code in your app or service, use [Manual Instrumentation](manual_instrumentation.md)
```

Comment:
```suggestion
Automatic instrumentation with Java uses a Java agent JAR that can be attached to any Java 8+ application. It dynamically injects bytecode to capture telemetry from many popular libraries and frameworks. It can be used to capture telemetry data at the "edges" of an app or service, such as inbound requests, outbound HTTP calls, database calls, and so on. To instrument application code in your app or service, use [Manual Instrumentation](manual_instrumentation.md)
```

I guess java agent isn't supposed to be preferred over library instrumentation, they're two good options